src/main/bin/ccm-fetch

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/perl ## no critic ((RequireVersionVar);
```

Comment:
this workaround is only needed for scripts (so i think 4 or 5 in all repos); and this is a bug in critic, see https://github.com/Perl-Critic/Perl-Critic/pull/711